test/index.js

```diff
@@ -28,8 +32,12 @@ describe('hermione-reassert-view', () => {
             }
         };
 
-        browser.addCommand = sinon.stub().callsFake((name, fn) => {
-            browser[name] = fn.bind(browser);
+        browser.$ = sinon.stub().named('$').resolves(element);
+        browser.overwriteCommand = sinon.stub().callsFake((name, command, isElement) => {
+            const target = isElement ? element : browser;
```

Comment:
Logic in tests should be as direct as possible without any conditions. So in that case you can write code like this:
```
sinon.stub()
  .withArgs(sinon.match.any, sinon.match.any, false).callsFake(...) // for stub browser
  .withArgs(sinon.match.any, sinon.match.any, true).callsFake(...) // for stub element
```

I know that we have the same code written in Hermione - https://github.com/gemini-testing/hermione/blob/master/test/lib/browser/utils.js#L83-L95. But in this plugin we can write a little better.